lib/versioning/aws-machine-image/index.spec.ts

```diff
@@ -43,9 +43,8 @@ describe('versioning/aws-machine-image/index', () => {
     });
   });
   describe('isGreaterThan(version1, version2)', () => {
-    it('should return false', () => {
-      expect(aws.isGreaterThan('ami-00', 'ami-99')).toBeFalse();
-      expect(aws.isGreaterThan('ami-99', 'ami-00')).toBeFalse();
+    it('should return true', () => {
+      expect(aws.isGreaterThan('', '')).toBeTruthy();
```

Comment:
```suggestion
    it('should always return true', () => {
      // https://github.com/renovatebot/renovate/discussions/13739
      expect(aws.isGreaterThan('ami-00', 'ami-99')).toBeTrue();
      expect(aws.isGreaterThan('ami-99', 'ami-00')).toBeTrue();
```

Reply:
Good point, added a more detailed explanation in my latest commit.